index.html

```diff
@@ -62,6 +62,142 @@ <h2 class="second-header">I'm a Software Developer</h2>
             <div class="main-image"></div>
         </section>
     </main>
+
+    <section class="work-section">
+        <h2 class="second-title">My recent Works</h2>
+        <hr class="line_below_title">
+        <div class="card-container">
+            <div class="main_card">
+                <img class="image_from_main_card" src="./assets/images/cardImages/mainCard.png" alt="Work 1">
+                <p class="main-card-title">Multi-Post Stories</p>
```

Comment:
- [ ] This title should use any of the `hx` tags because is semantically correct.